infra/modules/database/main.tf

```diff
@@ -6,7 +6,7 @@ locals {
   primary_instance_name = "${var.name}-primary"
   role_manager_name     = "${var.name}-role-manager"
   role_manager_package  = "${path.root}/role_manager.zip"
-
+  name                  = substr(var.name, 0, 12)
```

Comment:
What is the purpose of this change? I Don't see it mentioned in the PR description.

Reply:
This prevents longer workspace names from causing errors when other resources use it as a prefix.

Reply:
@charles-nava I just caught this. I'm concerned that this might cause namespace collisions. We use the naming convention: `<WORKSPACE>-<APP_NAME>-<ENVIRONMENT>-`, which means that the environment is going to be the portion that will get chopped off in longer names. I'm going to modify my PR #472 to propose reverting this `substr` change. Thoughts?

Reply:
@rocketnova agree. taking the prefix is similar to sweeping the dirt under the rug. instead we should let it fail early and solve the issue at the root